### PR TITLE
Adds recipe for canned laughter

### PR DESCRIPTION
**Pull request by GuyonBroadway**
Adds a chem recipe for canned laughter. Made with banana juice, space cola and space lube. 

Pointless but amusing, perhaps the clown could fill the pool with this stuff or make bright pink smoke grenades.

### CODE REVIEW

**Comment by ArcLumin** [comment]
/datum/chemical_reaction/drink/laugther

> laugther

I MEAN, IT WORKS

but yeah, typo lad. Still, code works as is.

**Comment by GuyonBroadway** [comment]
Fug, gimmie sec to fix typo

**Comment by ArcLumin** [comment]
Everything checks out. Code's good.

File: html/changelogs/GuyonBroadway - Recipie for Laghter.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a recipe for Laugher, the stuff that is found in the clowns can of canned laughter. Made with 1 part Banana Juice, one part Cola and one part Space lube."
```
Comment (McBawbaggings) [inline html/changelogs/GuyonBroadway - Recipie for Laghter.yml]: "Laugher". Also you change style on the recipe.

Comment (ghost) [inline html/changelogs/GuyonBroadway - Recipie for Laghter.yml]: "Laugher"
You also change style on the recipe itself by changing from 1 to one.

**Comment by ghost** [comment]
I was pointing out how you spelt it as "laugher" and I was hoping you would notice it's actually "laughter".

**Comment by ghost** [comment]
and oh yeah +1 since it's okay

**Comment by GuyonBroadway** [comment]
I also managed to totally cock up the commit message, _good job Guyon!_

All it does is fix the typo in the changelog.

**Comment by ghost** [comment]
ALL YOU'VE DONE IS ADDED A NEW CHANGELOG WITH A DIFFERENT NAME. Goodness me, just delete one changelog and fix the typo on the line.

**Comment by doobiedoo23** [comment]
aaaaaaaa god no canned laughter is the WORST

**Comment by SplergeBucket** [comment]
+1

**Comment by GuyonBroadway** [comment]
Sometimes I think that if I repeatedly threw sausages at the keyboard it would come out with less typos and more coherency overall than my typing.

**Comment by ghost** [comment]
Currently sitting at 1 + 2/3 of a vote.

**Comment by ArcLumin** [comment]
What do I count as, 1/3rd? I say okay to it.

**Comment by ArcLumin** [comment]
I count as +i because my input only has imaginary worth, rip.

**Comment by blobbernaut** [comment]
+1

**Comment by Swagile** [comment]
So whats the dealio on this?

Full approve and merge, more testing required, or no?

**Comment by JohnGinnane** [comment]
Need head input, unless Blobb's +1 counts in which case it'd be approved

**Comment by ghost** [comment]
No, blob doesn't count. Although doobie has resigned I believe his vote still counts.

**Comment by Swagile** [comment]
WAKE ME UP.

WAKE ME UP INSIDE.

CAN'T WAKE UP.

WAKE ME UP INSIDE.

SAAVVEE MEE!!!

**Comment by SplergeBucket** [comment]
bawb said +1 and i said +1 so it's good for merging???

**Comment by ghost** [comment]
Eh, whatever.

**Pull request merged**